Project-Hotpot/Project-Hotpot/Algorithm/SongAlgorithm.swift

```diff
@@ -9,9 +9,7 @@ import Foundation
 import Parse
 import AFNetworking
 
-class songAlgorithm{
-    var defaultSong: String = "spotify:album:3RQQmkQEvNCY4prGKE6oc5"
-    
+class songAlgorithm{    
     //TODO: CRASHES WHEN TOO MANY SONG REQUESTS, FOUND STACKOVERFLOW ALREADY
```

Comment:
If this is known, consider showing some kind of error screen to the user like "Too many requests, please wait for sometime and try again"